src/finite_elasticity_routines.f90

```diff
@@ -4008,6 +4031,14 @@ SUBROUTINE FINITE_ELASTICITY_GAUSS_CAUCHY_TENSOR(EQUATIONS_SET,DEPENDENT_INTERPO
           lambda_a=UP-(UP-LOW)/2.0_DP
 
           F_a_inv=0.0_DP
+          IF(lambda_a<TOL) THEN
+           LOCAL_ERROR="Warning: lambda_a is close to zero"
+           CALL FlagError(LOCAL_ERROR,ERR,ERROR,*999)
+!            WRITE(*,*) "UP: ", UP
```

Comment:
P.S. Thanks a lot. Hope you had a merry X-mas, too. All the best for the future! Hope your ankle gets well soon.